src/app/api/notes/new/route.ts

```diff
@@ -0,0 +1,37 @@
+import prisma from "@/utils/prisma";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(request: NextRequest) {
+    const userID = 1; // TODO need to be retrieved from the token
+    if (request.body) {
```

Comment:
Empty requests are not stopped by the
```ts
if (request.body) {}
```
And always return a 500 error, meaning there is a problem server side.